src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -73,81 +60,95 @@ You can view a deeper layer of granularity in your metrics by clicking on any me
+A common method to maintain account cardinality within its limit is to implement pruning rules for non-critical attributes, which removes unnecessary data.
 
 ### View account pruning rules [#view-pruning-rules]
 
-You can view pruning rules by clicking into a metric and then clicking **View Pruning Rules**. This will bring up a scrollable list of all pruning rules associated with an account. You can also delete pruning rules associated with an account in this view:
-
+To view account pruning rules:
+1. Go to the **Cardinality Management** page of the account.
+2. From the **Metrics** table, select <DNT>**View Pruning Rules**</DNT> to get a scrollable list of all pruning rules associated with the account.
+  <Callout variant="tip">
+If no pruning rules exist in the account, the <DNT>**View Pruning Rules**</DNT> button doesn't appear.
+</Callout>
 <img
     style={{ align: 'left',maxWidth: '90%' }}
     title="Screenshot showing the link to pruning rules on the Attributes page"
     alt="Screenshot showing the link to pruning rules on the Attributes page"
     src="/images/apis-and-data_screenshot-crop_view-pruning-rules.webp"
 />
 
-
 ### Create a pruning rule [#create-pruning-rule]
 
-You can create a pruning rule based on attributes within the UI. To create a pruning rule:
+To create a pruning rule:
 
-1. Select the checkboxes next to the attributes you want to prune. 
+1. Go to the **Cardinality Management** page of the account.
+2. From the **Metrics** table, select the metric in which you want to create a pruning rule. The **Cardinality Management** page of the metric opens.
+3. Go to the **Attributes** table and select the attributes you want to prune. 
   <img
     style={{ align: 'left', maxWidth: '90%' }}
     title="Screenshot showing how to select attributes for pruning rule"
     alt="Screenshot showing how to select attributes for pruning rule"
     src="/images/apis-and-data_screenshot-crop_select-attributes-for-pruning-rule.webp"/>
 
-2. Click **Create pruning rule**.
-3. In the pop-up with information about how the pruning rule affects the metric's cardinality, enter a rule name or description.
+3. Select <DNT>**Create pruning rule**</DNT>. A pop-up appears, providing details on how the pruning rule impacts the metric's cardinality.
+  
   <img
     style={{ align: 'left' }}
     title="Screenshot of page to name the pruning rule"
     alt="Screenshot of page to name the pruning rule"
     src="/images/apis-and-data_screenshot-crop_name-pruning-rule.webp"/>
-    
-4. Click **Submit** to generate the pruning rule. 
 
-It's important to note that pruning rules are instantaneous. The information will be pruned instantly after creating the rule, so make sure you really want to prune the data. 
+3. Enter a rule name or description.
+    
+4. Select <DNT>**Submit**</DNT> to create the pruning rule.
```

Comment:
To create....
Is select the right word here?

Reply:
Updated it as following:
To create the pruning rule, click <DNT>**Submit**</DNT>.